datadog/__init__.py

```diff
@@ -41,9 +41,9 @@ def initialize(api_key=None, app_key=None, host_name=None, api_host=None,
     :param app_key: Datadog application key
     :type app_key: string
 
-    :param proxies: Proxy to use to connect to Datadog API
+    :param proxies: Proxy to use to connect to Datadog API; \
```

Comment:
I don't know how backslashes render, you can just do:
```suggestion
    :param proxies: Proxy to use to connect to Datadog API;
```